### PR TITLE
[Testing required] Allows you to actually win the game again.

### PR DESCRIPTION
**Pull request by Kmc2000**
## Why It's Good For The Game

Maybe winning was the friends we made along the way

## Changelog
:cl:
fix: Win condition now works again, good hunting!
/:cl:

### CODE REVIEW

**Comment by theboy6545** [comment]
Hoc modo oparandor.

File: nsv13/code/modules/overmap/types/nanotrasen.dm [hunk 1]
```diff
@@ -134,7 +134,7 @@
 	armor = list("overmap_light" = 75, "overmap_heavy" = 35)
 	pixel_w = -44
 	pixel_z = -180
-	starting_system = "Unknown Signal"
+	starting_system = "Wolf 359"
```
Comment (ike709) [inline nsv13/code/modules/overmap/types/nanotrasen.dm:137]: OwO

**Review by Vasily2013 — APPROVED** [review]
Admins report this is working. Gonna merge before KMC turns it into an omnibus.

**Comment by thudner24** [comment]
The message that says the crew can return to risa showed up again after we defeated the sixth fleet, so i assume that happens every three fleets.

**Pull request merged**